### ISSUE TITLE
nuclinfo.tors() not converting delta to 0-360°

### ISSUE THREAD

**Issue by nestorwendt** [MDAnalysis/mdanalysis#1572]
### From MDAnalysis/analysis/nuclinfo.py

```
    alpha = a.dihedral.value()
    beta = b.dihedral.value()
    gamma = g.dihedral.value()
    delta = d.dihedral.value()
    epsilon = e.dihedral.value()
    zeta = z.dihedral.value()
    chi = c.dihedral.value()

    if alpha < 0:
        alpha = alpha + 360
    if beta < 0:
        beta = beta + 360
    if gamma < 0:
        gamma = gamma + 360
    if epsilon < 0:
        epsilon = epsilon + 360
    if zeta < 0:
        zeta = zeta + 360
    if chi < 0:
        chi = chi + 360
    return [alpha, beta, gamma, delta, epsilon, zeta, chi]
```

Delta conversion is missing. I am working on it.

### Current version of MDAnalysis:
0.16.2

**Comment by mnmelo** [comment]
I edited your initial comment to fix the version you're using (you had '1.6.2', changed it to '0.16.2').

**Comment by mnmelo** [comment]
Closed by 0f9ef86
